Formula/haxe.rb

```diff
@@ -1,9 +1,8 @@
 class Haxe < Formula
   desc "Multi-platform programming language"
   homepage "https://haxe.org/"
-  url "https://github.com/HaxeFoundation/haxe.git",
-      :tag      => "4.0.5",
-      :revision => "67feacebc59d5cd9dce232058f12cb933622d619"
+  url "https://github.com/HaxeFoundation/haxe/archive/4.1.0.tar.gz"
+  sha256 "c815b0671b0a679faab226464fe4f75c9868c5084a39073f6698ebc27e1455b1"
```

Comment:
We need to use `:tag` because there is git submodules in the Haxe repo. Git submodules are absent from the GitHub archive.

Reply:
TIL thanks!

Reply:
It built successfully on a local machine, let's see what CI thinks about it. 🤞